modules/Search/components/Hit.tsx

```diff
@@ -22,9 +22,14 @@ function HitComponent({ hit }: Props) {
     const { categories } = story;
     const { showDate, showSubtitle } = useThemeSettings();
 
+    const storyLink = {
```

Comment:
Please add a comment indicating why this change was needed. That will protect this from future refactorings :)

Reply:
Added the comment!